ContactDetailsApi/V2/Infrastructure/ContactUprns.cs

```diff
@@ -0,0 +1,35 @@
+using Hackney.Shared.Person.Infrastructure;
+using Hackney.Shared.Tenure.Domain;
+using System;
+using System.Collections.Generic;
+
+namespace ContactDetailsApi.V2.Infrastructure
+{
+    public class ContactByUprn
+    {
+        public string Uprn { get; set; }
+        public Guid? TenureId { get; set; }
+        public List<Person> Contacts { get; set; }
+
+    }
+
+
+    public class Person
```

Comment:
I wanted to keep it in one class as I didn't want it to be confusing with the other shared objects we have. These objects are only specifically for this endpoint and i don't think it would be used be other endpoints